pkg/cloudprovider/provider/gce/config.go

```diff
@@ -123,32 +127,41 @@ func newConfig(resolver *providerconfig.ConfigVarResolver, spec v1alpha1.Provide
 	if err != nil {
 		return nil, err
 	}
+
 	// Setup configuration.
 	cfg := &config{
 		providerConfig: providerConfig,
+		labels:         cpSpec.Labels,
+		diskSize:       cpSpec.DiskSize,
+		network:        cpSpec.Network.Value,
```

Comment:
Why not using the resolver like below? Any hint is welcome to learn more about our environment. 😅

Reply:
Resolver will try to look for secret/configmap and once it see there is no such thing for net/subnet it will use .Value as fallback. So I've used that property %)

But you're right, those now are resolved properly and could contain secret/configmap references in spec.